prac_09/band.py

```diff
@@ -0,0 +1,19 @@
+class Band:
+    """Band class."""
+
+    def __init__(self, name=""):
+        """Construct a Band with a name and empty members collection."""
+        self.name = name
+        self.members = []
+
+    def __str__(self):
+        """Return a string representation of a Band."""
+        return f"{self.name} ({','.join([str(member) for member in self.members])})"
+
+    def play(self):
+        """Return a string showing the members performance."""
+        return "\n".join([member.play() for member in self.members])
+
+    def add(self, member):
+        """Add members."""
+        self.members.append(member)
```

Comment:
Your method name '_add_' is good, but it's always a good practice to ensure consistency.  You might want to consider using a consistent naming convention, like '_add_member'_ instead! 🌮